packages/db/postgres/migrations/023-add-user-customer-table.sql

```diff
@@ -0,0 +1,9 @@
+-- add table to keep track of 0..1 customer id for each user
+CREATE TABLE IF NOT EXISTS user_customer
+(
+  id               BIGSERIAL PRIMARY KEY,
+  user_id          BIGINT NOT NULL UNIQUE REFERENCES public.user (id),
+  customer_id      TEXT NOT NULL UNIQUE
+);
+
+CREATE INDEX IF NOT EXISTS user_customer_user_id_idx ON public.user_customer (user_id);
```

Comment:
```suggestion
CREATE INDEX CONCURRENTLY IF NOT EXISTS user_customer_user_id_idx ON public.user_customer (user_id);
```

In migration we should run concurrently to not lock anything. In this case, table is not existent yet, so should not be a problem, but also for future reference when adding indexes on already live things